frontend/packages/dsco/src/divider/divider.module.scss

```diff
@@ -0,0 +1,39 @@
+@import '@/common/styles/primitiveColors.css';
+
+// Divider common styles
+hr.divider {
```

Comment:
is there a reason why this rule is `hr.divider` instead of just `.divider`?
In case consumer will want to add custom styles it'll be harder to override `hr.divider` styles than it is to override `.divider`

Reply:
Nope, I find it clearer to include the HTML tag too, but I can remove it!